LICENSE-3rdparty.csv

```diff
@@ -172,6 +172,18 @@ go.sum,github.com/yuin/goldmark,MIT,2019 Yusuke Inuzuka
 go.sum,golang.org/x/mod,BSD-3-Clause,2009 The Go Authors.
 go.sum,google.golang.org/protobuf,BSD-3-Clause,2018 The Go Authors.
 go.sum,gopkg.in/errgo.v2,BSD-3-Clause,"Copyright © 2013, Roger Peppe"
-go.sum,gopkg.in/warnings.v0,BSD 2-Clause "Simplified" License,2016 Péter Surányi.
+go.sum,gopkg.in/warnings.v0,BSD 2-Clause,2016 Péter Surányi.
 go.sum,rsc.io/quote/v3,BSD-3-Clause,2009 The Go Authors.
 go.sum,rsc.io/sampler,BSD-3-Clause,2009 The Go Authors.
+go.sum,github.com/Masterminds/goutils,Apache-2.0,Copyright 2014 Alexander Okoli
+go.sum,github.com/Masterminds/semver,MIT,"2014-2019, Matt Butcher and Matt Farina"
+go.sum,github.com/Masterminds/sprig,MIT,2013-2020 Masterminds
+go.sum,github.com/andybalholm/crlf,BSD-2-Clause,2015 Andy Balholm.
+go.sum,github.com/apparentlymart/go-textseg/v12,<LICENSE>,<COPYRIGHT>
```

Comment:
Can we update the license and copyright info here.

Reply:
Updated